src/python/pants/backend/go/goals/test.py

```diff
@@ -104,6 +105,12 @@ def opt_out(cls, tgt: Target) -> bool:
         return tgt.get(SkipGoTestsField).value
 
 
+class GoTestRequest(TestRequest):
+    # TODO: Remove the type-ignore after adding a `skip` option to the subsystem.
+    tool_subsystem = GoTestSubsystem  # type: ignore[assignment]
```

Comment:
lol

Reply:
Yeah - would be just as many LoC to actually implement the skip options as part of this, but trying to keep the plugin API changes and user API changes separate...

Reply:
Opened #17338